cg/meta/workflow/nextflow_common.py

```diff
@@ -86,13 +86,37 @@ def create_samplesheet_csv(
 
     @classmethod
     def get_verified_arguments_nextflow(
-        cls, case_id: str, log: Path, pipeline: str, root_dir: str
+        cls, case_id: str, pipeline: str, root_dir: str, log: Path, bg: bool, q: bool
```

Comment:
What is `bg` and `q`? Is there a more mnemonic name for these args?

Reply:
good point! 
I can change `q` for `quiet`. `bg` needs to stay like that and is for "executing in the background"

Reply:
How about: `use_child_process` or `use_background_process` at least for the python arg. I do not know if nextflow has a long option for `bg`. In code always use the long option as it is more clear when reading/executing.